docs/public-networks/reference/api/objects.md

```diff
@@ -212,7 +212,7 @@ and
 
 ## Transaction call object
 
-Parameter for [`eth_call`](index.md#eth_call) and
+Parameters for [`eth_call`](index.md#eth_call), [`eth_createAccessList`](index.md#eth_createAccessList) and
```

Comment:
```suggestion
Parameter for [`eth_call`](index.md#eth_call), [`eth_createAccessList`](index.md#eth_createAccessList), and
```

"Parameter" seems to make more sense here since it refers to the single transaction call object. The same is used for the filter options object earlier on this page.